core/shared/src/main/scala/scalaz/zio/Schedule.scala

```diff
@@ -240,16 +240,27 @@ trait Schedule[-A, +B] extends Serializable { self =>
     }
 
   /**
-   * An alias for `<||>`
+   * Alias for andThenEither.
+   *
+   * Returns a new schedule that first executes this schedule to completion,
+   * and then executes the specified schedule to completion.
    */
-  final def andThen[A1 <: A, C](that: Schedule[A1, C]): Schedule[A1, Either[B, C]] =
-    self <||> that
+  final def <||>[A1 <: A, C](that: Schedule[A1, C]): Schedule[A1, Either[B, C]] =
```

Comment:
Let's delete this symbolic name. It isn't used widely enough to matter.

Reply:
:heavy_check_mark: